src/sentry/utils/event_tracker.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import random
+from enum import Enum
+
+
+class EventStageStatus(Enum):
+    START = "start"
+    END = "end"
+    REDIS_PUT = "redis_put"
+    """
+    i plan on adding the below enums for every step of the transactions pipeline
+    ingest_consumer_published
+
+    redis_put
+
+    save_event_started
+
+    save_event_finished
+
+    snuba_topic_put
+
+    commit_log_topic_put
+
+    ppf_topic_put
+
+    post_process_started
+
+    post_process_finished / the same as redis_deleted
+    """
+
+
+class EventTracker:
```

Comment:
I think you can probably just have a couple functions here, you don't need a class for this